### PR TITLE
Performance tests

### PR DESCRIPTION
**Pull request by mminutoli**

### CODE REVIEW

**Comment by mminutoli** [comment]
@tjstavenger-pnnl I started this pull request to give you feedback and comments.

However, I think we are close to merge it to master.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 *~
 build*
+/.vscode/
```
Comment (mminutoli) [inline .gitignore]: Please put a newline at the end of the file.  It might be better to have your editor setup to do it for you.

File: README.md [hunk 2]
```diff
@@ -48,7 +62,8 @@ cmake .. -DCMAKE_INSTALL_PREFIX=$SHADROOT \
          -DSHAD_RUNTIME_SYSTEM=TBB        \
          -DTBB_ROOT=$TBBROOT              \
          -DGTEST_ROOT=$GTESTROOT          \
-         -DGPERFTOOLS_ROOT=$GPERFTOOLSROOT
+         -DGPERFTOOLS_ROOT=$GPERFTOOLSROOT \
```
Comment (mminutoli) [inline README.md]: Can you please align all the "\" characters?

Sorry, I am kind of affected by OCD :-P

**Comment by tjstavenger-pnnl** [comment]
@mminutoli -- do you have a preference for how we add a new `perf_test` target? I had wanted to do that, but didn't find a great way to dynamically add each of the tests performance tests into the new target (like you can with `add_test()`). The cmake `add_custom_target()` requires that you know all of the commands to execute when it is getting called, but right now we don't know that since the performance tests are split between two folders.

**Comment by mminutoli** [comment]
I see your point but I don't have an answer for that 😄.  I'll think about it and I'll talk with @VitoCastellana

**Comment by mminutoli** [comment]
@VitoCastellana : This should be ready for merge.

**Comment by mminutoli** [comment]
@VitoCastellana: @tjstavenger-pnnl is the author; I did the review and created the pull request, but I can't merge it because I initiated the pull request.  Can you hit the merge button for me?

**Comment by VitoCastellana** [comment]
@tjstavenger-pnnl Tim, can you please fix the conflict on the travis.yml file?
We can merge as soon as the conflict is resolved.

**Comment by tjstavenger-pnnl** [comment]
Conflicts resolved & Travis build passed at https://travis-ci.org/pnnl/SHAD/builds/382851037

**Pull request merged**